### ISSUE TITLE
goss version incompatible with Packer

### ISSUE THREAD

**Issue by kbreit** [kubernetes-sigs/image-builder#560]
**What steps did you take and what happened:**
```
[kbreit@e4cent0dev1 capi]$ make deps-ova
...
[kbreit@e4cent0dev1 capi]$ make build-node-ova-vsphere-clone-ubuntu-2004
hack/ensure-ansible.sh
hack/ensure-ansible-windows.sh
hack/ensure-packer.sh
hack/ensure-goss.sh
Right version of binary present
hack/ensure-ovftool.sh
packer build -var-file="/home/kbreit/image-builder/images/capi/packer/config/kubernetes.json"  -var-file="/home/kbreit/image-builder/images/capi/packer/config/cni.json"  -var-file="/home/kbreit/image-builder/images/capi/packer/config/containerd.json"  -var-file="/home/kbreit/image-builder/images/capi/packer/config/ansible-args.json"  -var-file="/home/kbreit/image-builder/images/capi/packer/config/goss-args.json"  -var-file="/home/kbreit/image-builder/images/capi/packer/config/common.json"  -color=true -var-file="packer/ova/packer-common.json" -var-file="/home/kbreit/image-builder/images/capi/packer/ova/ubuntu-2004.json" -var-file="packer/ova/vsphere.json" -except=esx -except=local -only=vsphere-clone  packer/ova/packer-node.json
Error: Failed to initialize build "vsphere-clone"

error initializing provisioner 'goss': The protocol of this plugin (protocol
version 4 and lower) was deprecated, please use a newer version of this
plugin.Or use an older version of Packer (pre 1.7) with this plugin.
```

**What did you expect to happen:**
I expected it to complete the build as it installed the packer-goss version.

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

Project ([Image Builder for Cluster API](https://github.com/kubernetes-sigs/image-builder/tree/master/images/capi), [kube-deploy/imagebuilder](https://github.com/kubernetes-sigs/image-builder/tree/master/images/kube-deploy/imagebuilder), [konfigadm](https://github.com/kubernetes-sigs/image-builder/tree/master/images/konfigadm)): CAPI

Additional info for Image Builder for Cluster API related issues: CentOS 8, Packer 1.7.0, Ansible 2.10.7

- OS (e.g. from `/etc/os-release`, or `cmd /c ver`):
- Packer Version:
- Packer Provider:
- Ansible Version: 
- Cluster-api version (if using):
- Kubernetes version: (use `kubectl version`):

/kind bug
[One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels]

**Comment by perithompson** [comment]
I found this as well, I bumped the packer version in my WIP PR  [#563](https://github.com/kubernetes-sigs/image-builder/pull/563) to 1.7 and that worked ok

**Comment by codenrhoden** [comment]
/close

The latest commits are using Packer v1.7.2, and a compatible version of the GOSS provisioner. Fixed with #570
